src/libs/sfdk/asynchronous.cpp

```diff
@@ -121,6 +137,7 @@ void CommandQueue::cancelBatch(BatchId batchId)
 void CommandQueue::enqueue(std::unique_ptr<CommandRunner> &&runner)
 {
     QTC_ASSERT(runner, return);
+    QTC_CHECK(!CommandRunner::isPostprocessing()); // use enqueueImmediate for postprocessing!
```

Comment:
I like the idea! :)

Reply:
Yeah, hopefully it is sufficient substitution for the missing documentation :) thanks! :)